tests/components/__snapshots__/user_settings_display.test.jsx.snap

```diff
@@ -340,6 +340,143 @@ exports[`components/user_settings/UserSettingsDisplay should match snapshot, clo
                   id="user.settings.display.preferTime"
                   values={Object {}}
                 />
+<<<<<<< HEAD
```

Comment:
You would have to resolve conflicts here in snapshots as well. Choose head or your commit changes appropriately.

Reply:
@sudheerDev , I have raised another PR with commits, Please check https://github.com/mattermost/mattermost-webapp/pull/398#issuecomment-348964883.
This is a closed PR

Reply:
@panditsavitags Oops. I just happened to see this commit So, was just pitching in.  I am not a reviewer. Let me see if I can help though :)

Reply:
:) thanks